app/views/internal/configs/show.html.erb

```diff
@@ -11,16 +11,74 @@
   </h2>
 </div>
 
+<div class="row my-3" id="getStarted">
+  <div class="card w-100">
+    <div class="card-header" id="getStartedHeader" data-toggle="collapse"
+      data-target="#getStartedBodyContainer" aria-expanded="true" aria-controls="getStartedBodyContainer">
+      <h2 class="d-inline">Get Started</h2>
+      <button class="btn btn-secondary float-right" type="button">
+        Toggle Get Started
+      </button>
+    </div>
+
+    <div id="getStartedBodyContainer" class="collapse show hide p-3" aria-labelledby="siteConfigHeader">
```

Comment:
This section is a `Get Started` section that will be expanded on the Site Config page. We will list the required fields in this section so a new forem user will easily and very quickly be able to configure their site without expanding and collapsing all those tabs. This will translate into an onboarding section for the forem admin at some point.

Reply:
Also we can probably move this into a partial. Once I get feedback and I'm polishing I can do this.

Reply:
Maybe with a small change we can also reuse `setup_completed?` from `VerifySetupCompleted` and hide this section if everything's been configured already?

Reply:
@Ridhwana This is one of the few cases where I think [`module_function`](http://ruby-doc.org/core-2.7.1/Module.html#module_function-method) is the best solution. It makes a method's mixin also available on the class level, so if you add `module_function :setup_completed?` to `VerifySetupCompleted` we're able to do this:

```ruby
VerifySetupCompleted.setup_completed?
```

Reply:
@citizen428 I was really scratching my head for this one so thank you, this was helpful!